MoppApp/MoppApp/ContainerActions.swift

```diff
@@ -52,7 +52,7 @@ extension ContainerActions where Self: UIViewController {
                 navController.setViewControllers([navController.viewControllers.first!], animated: false)
              
                 let ext = urls.first!.pathExtension
-                if (ext.isContainerExtension || ext == ContainerFormatPDF) && urls.count == 1 {
+                if ((ext.isAsicContainerExtension || ext == ContainerFormatPDF) && landingViewController.containerType == .asic) && urls.count == 1 {
```

Comment:
If there is a long line with lots of chained conditions it would be preferable to write it in multiple lines, with each line ending with the logical operator (&&, || etc).
```swift
if ((ext.isAsicContainerExtension || ext == ContainerFormatPDF) && 
    landingViewController.containerType == .asic) && 
    urls.count == 1 {
    ...
    }
```